packages/server/src/cli.ts

```diff
@@ -47,6 +49,8 @@ if (isCli) {
 		readonly host: string;
 		readonly port: number;
 
+		readonly extensionsDir?: string;
+		readonly userDataDir?: string;
```

Comment:
`extensionsDir` and `userDataDir` are unused?

Reply:
Oh right. Going to add this in another PR.